### PR TITLE
Table preview

### PR DESCRIPTION
**Pull request by Halama**
Kromě preview tabulky jsou odstraněný taky dlouho deprecated classy a metody.
Bude potom releasnuto jako major verze `7.0.0`.

### CODE REVIEW

File: apiary.apib [hunk 1]
```diff
@@ -1186,10 +1186,11 @@ Further information can be found in the [Developers Documentation](https://devel
             }
 
 
-## Unload Data [/v2/storage/tables/{table_id}/export]
-### Export table into CSV [GET]
+## Data Preview [/v2/storage/tables/{table_id}/data-preview]
+### Data preview [GET]
 
-This method is **DEPRECATED**, please use the [asynchronous version](#reference/tables/unload-data-asynchronously) of this method.
+Returns up to 1000 rows from table in CSV format. Rows can be filtered by various filters described below.
```
Comment (pivnicek) [inline apiary.apib]: How about using `limit` and `offset` rather than a hard 100.  could maybe put a limit on limit of 1000

Comment (Halama) [inline apiary.apib]: I don't know may when there'll be some use case for `offset`

Comment (Halama) [inline apiary.apib]: offset makes only sense with sort order defined etc. 
We just want a preview working asap and turn off sync exports of large tables.

Comment (pivnicek) [inline apiary.apib]: fair enough:).  it's always something that can be added later during console work

Comment (Halama) [inline apiary.apib]: yes,  I am thinking about some sorting or random sort for preview use cases in console.

Comment (odinuv) [inline apiary.apib]: i think both random sort and some defined sort make sense for preview

**Pull request merged**